internal/filtering/blocked.go

```diff
@@ -374,6 +374,19 @@ var servicesData = []svc{{
 		"||dreamcast.hk^",
 		"||hdslb.com^",
 	},
+}, {
+	name: "globo",
+	rules: []string{
+		"||globoplay.globo.com^",
+		"||globoplay.com^",
+		"||globoplay.com.br^",
+		"||canaisglobo.globo.com^",
+		"||premiere.globo.com^",
+		"||canaisglobo.globo.com^",
+		"||glbimg.com^",
+		"||cloud-jarvis.globo.com^",
+		"||api-soccer.globosat.tv^",
```

Comment:
Please sort the rules.

Also, pay attention to the comment on `servicesData`.  `client/src/components/ui/Icons.js` must be updated as well.